scripts/border_defend_station.lua

```diff
@@ -33,6 +37,34 @@ function init()
 	--name, faction _id and template set by calling script
 	my_ship = CpuShip():setCallSign(string.format("%s %s",station_name,name)):setCommsScript(""):setCommsFunction(commsDefendShip):setFactionId(faction_id):setPosition(position_x, position_y):setTemplate(template):setScanned(true):orderDefendTarget(my_station)
 end
+function isObjectType(obj,typ)
```

Comment:
Wouldn't it be better to use the new generic function you introduced? I merged that one.

Reply:
Yes it would. At the time that I made this pull request, I had not made the generic utility yet. I made a bunch of other Lua file changes with their own internal function before the light bulb went off about making a generic routine for other Lua code to use. I have not gone back to those other Lua document pull requests to use the utility yet.